PokerHandTestProject/PokerHandTestCases.cs

```diff
@@ -19,17 +19,26 @@ public void PokerHandInputValidTestEvaluation(string PokerHandValue, string expe
             //act and assert
             Assert.Equal(expectedOutput, (PKObj.ValidatePokerHandInput(PokerHandValue)));
         }
+       // [Fact]
+       // //[InlineData("Ac Kc Qc Jc Tc", "Valid Data")]
+       //// [InlineData(string.Empty)]
+       // public void PokerHandInputValidTestEvaluation1()
+       // {
+       //      ArgumentException Exception = new ArgumentException();
+       //     //act and assert
+       //     Assert.Equal(Exception.Message, (PKObj.ValidatePokerHandInput(null)));
+       // }
         [Theory]
         [InlineData("Ac Kc Qc Jc Tc", "Royal Flush")]
-        [InlineData("Ac As Jc Tc Ts", "")]
+        [InlineData("Ac As Jc Tc Ts", "No Match")]
         public void PokerHandRoyalFlushEvaluation(string PokerHandValue, string expectedOutput)
```

Comment:
you are doing multiple things and thus the test holds no value. 
PokerHandRoyalFlushEvaluation, but there is output of No Match?